natives.json

```diff
@@ -111630,7 +111630,7 @@
 					"name": "p12"
 				}
 			],
-			"return_type": "Any",
+			"return_type": "void",
```

Comment:
Please do not change any return type to ``void``. The actual return type is ``BOOL``.